packages/celery-library/requirements/_base.in

```diff
@@ -0,0 +1,11 @@
+#
+# Specifies third-party dependencies for 'celery-library'
+#
+--constraint ../../../requirements/constraints.txt
+--requirement ../../../packages/common-library/requirements/_base.in
+--requirement ../../../packages/models-library/requirements/_base.in
+--requirement ../../../packages/service-library/requirements/_base.in
```

Comment:
if would be nice if you avoid service-library all the way. Why would you need it?